src/Bulk.php

```diff
@@ -174,4 +174,13 @@ public function flush()
         $this->itemList = [];
         $this->itemCount = 0;
     }
+
+    /**
+     * Clears all items in the Bulk object.
+     */
+    public function clear()
+    {
+        $this->itemList = [];
+        $this->itemCount = 0;
+    }
```

Comment:
Now that this function exists, it should really be called in places where the code currently explicitly clears the item list. (e.g. line 174 & 175 above).